src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -105,23 +128,33 @@ public BraveTracer build() {
     }
   }
 
-  final brave.Tracer brave4;
   final Map<Format<TextMap>, Injector<TextMap>> formatToInjector = new LinkedHashMap<>();
   final Map<Format<TextMap>, Extractor<TextMap>> formatToExtractor = new LinkedHashMap<>();
 
   BraveTracer(Builder b) {
     brave4 = b.brave4.tracer();
+    activeSpanSource = b.activeSpanSource;
```

Comment:
if check null in builder, you can do this safely here as there's no other way to assign the value.